src/transformers/models/mbart/tokenization_mbart50_fast.py

```diff
@@ -273,6 +273,15 @@ def set_tgt_lang_special_tokens(self, tgt_lang: str) -> None:
             special_tokens=list(zip(prefix_tokens_str + suffix_tokens_str, self.prefix_tokens + self.suffix_tokens)),
         )
 
+    def _build_translation_inputs(self, raw_inputs, src_lang: Optional[str], tgt_lang: Optional[str], truncation):
+        if src_lang is None or tgt_lang is None:
```

Comment:
we should add this method in all many to many tokenizes (`MBart50Tokenizer`, `M2M100Tokenizer`)

Reply:
Done.